Tools/clinic/clinic.py

```diff
@@ -1981,11 +1984,11 @@ def __getitem__(self, i):
             self._array.append(self._constructor())
         return self._array[i]
 
-    def clear(self):
+    def clear(self) -> None:
         for ta in self._array:
-            ta._text.clear()
+            ta.text.clear()
```

Comment:
Oh, was this a bug? Should we add a regression test?

Reply:
Yes, it was. I'll try to produce a test that triggers it.